src/auxinfo/participant.rs

```diff
@@ -138,6 +139,8 @@ impl ProtocolParticipant for AuxInfoParticipant {
         other_participant_ids: Vec<ParticipantIdentifier>,
         input: Self::Input,
     ) -> Result<Self> {
+        setup_zeroize();
```

Comment:
That makes sense. This can also be an optional feature.

It should be called before anything else though. Otherwise it probably still works but we won’t guarantee it.